dask/dataframe/utils.py

```diff
@@ -671,11 +671,14 @@ def check_matching_columns(meta, actual):
     if not np.array_equal(np.nan_to_num(meta.columns), np.nan_to_num(actual.columns)):
         extra = actual.columns.difference(meta.columns).tolist()
         missing = meta.columns.difference(actual.columns).tolist()
+        if extra or missing:
+            extra_info = f"  Extra:   {extra}\n  Missing: {missing}"
```

Comment:
Is the added space after the quote intentional ?

Reply:
Yeah, that's how it was in the original. Black just made it go all on one line.